src/Security/Authorization/Policy/src/PublicAPI.Shipped.txt

```diff
@@ -1,6 +1,6 @@
 #nullable enable
 Microsoft.AspNetCore.Authorization.AuthorizationMiddleware
-Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.AuthorizationMiddleware(Microsoft.AspNetCore.Http.RequestDelegate! next, Microsoft.AspNetCore.Authorization.IAuthorizationPolicyProvider! policyProvider) -> void
+Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.AuthorizationMiddleware(Microsoft.AspNetCore.Http.RequestDelegate! next, Microsoft.AspNetCore.Authorization.IAuthorizationPolicyProvider! policyProvider, Microsoft.Extensions.Logging.ILogger<Microsoft.AspNetCore.Authorization.AuthorizationMiddleware!>! logger) -> void
```

Comment:
Previously shipped APIs can't be modified. You need to add new overloads instead.

Reply:
Okay, thank you for the tip, I wasn't sure how to handle that😅